tests/bench_runtime_coreapis/README.md

```diff
@@ -0,0 +1,7 @@
+# Measure Runtime of Selected Core API functions
+
+This benchmark application measures the runtime of selected core API functions.
+Its purpose is to provide a baseline to assess the impacts when doing changes to
+core code.
```

Comment:
Re this - would it be worth raising an issue/opening another PR which aims to do this globally? I.e. at the moment the actual data isn't automatically collected and used to assess the impacts of changes, it's a tool for developers to use. Would it be worth considering and documenting extension work at this stage? Also including expanding the testing, and creating documentation to point users towards these tests?

Reply:
I don't think we want to do that at this point, the work on automating this is not mature enough and we also don't have enough experience to go into process mode yet... Also this is not directly a concern of this PR and should be discussed/considered somewhere else :-)

Reply:
ok fair enough